content/webapp/components/StoryPromo/StoryPromo.tsx

```diff
@@ -37,9 +37,9 @@ const StoryPromo: FunctionComponent<Props> = ({
 
   const seriesColor = seriesWithSchedule?.color ?? undefined;
 
-  const indexInSeriesSchedule = (seriesWithSchedule?.schedule
-    ?.map(scheduleItem => scheduleItem.title) || [])
-    .indexOf(article.title);
+  const indexInSeriesSchedule = (
+    seriesWithSchedule?.schedule?.map(scheduleItem => scheduleItem.title) || []
+  ).indexOf(article.title);
```

Comment:
Intermediate variable for readability:

```suggestion
  const seriesTitles = seriesWithSchedule?.schedule?.map(scheduleItem => scheduleItem.title) || [];
  const indexInSeriesSchedule = seriesTitles.indexOf(article.title);
```